### PR TITLE
feat: nav refactor

### PR DESCRIPTION
**Pull request by SpencerKaiser**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

## Description of Changes

Refactoring the nav with better usability

## Related Issues

https://github.com/AmericanAirlines/Hangar/assets/6445731/7d0e4b51-6b15-4733-96fb-687f66715b03

https://github.com/AmericanAirlines/Hangar/assets/6445731/0105cec5-6180-4c6c-bc09-e4a74de80853

### CODE REVIEW

File: packages/web/src/pages/schedule.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ const Schedule: NextPage = () => {
   }, [toast]);
 
   return (
-    <PageContainer pageTitle={'Schedule'} heading={'Events'}>
+    <PageContainer pageTitle={'Schedule'} heading={'Schedule'}>
```
Comment (granthill7) [inline packages/web/src/pages/schedule.tsx:47]: oopsies my bad :)

File: packages/web/src/components/layout/AppLayout/NavBar/NavDrawer.tsx [hunk 2]
```diff
@@ -1,91 +1,59 @@
...
-                  onClick={() => {
-                    window.location.href = '/api/auth/logout';
-                  }}
-                >
-                  {`Logout`}
-                </Button>
-              )}
+              <Schedule />
+
+              {admin && <AdminDashboard />}
```
Comment (granthill7) [inline packages/web/src/components/layout/AppLayout/NavBar/NavDrawer.tsx:51]: This just says "If admin exists, show <AdminDashboard /> right?"

File: packages/web/src/pages/schedule.tsx (hunk 1 above)
Comment (SpencerKaiser) [reply packages/web/src/pages/schedule.tsx:47]: This wasn't you and I originally saw it during code review but thought I'd leave it; it later started to bug me so I changed it

File: packages/web/src/components/layout/AppLayout/NavBar/NavDrawer.tsx (hunk 2 above)
Comment (SpencerKaiser) [reply packages/web/src/components/layout/AppLayout/NavBar/NavDrawer.tsx:51]: Yessir!

**Pull request merged**